charts/fluentd-elasticsearch/templates/prometheusrule.yaml

```diff
@@ -11,77 +11,9 @@ metadata:
     {{- end }}
   namespace: {{ .Values.prometheusRule.prometheusNamespace }}
 spec:
+  {{- with .Values.prometheusRule.rules }}
```

Comment:
imho this will not work as there are variables used inside the rule.
tpl function could be used instead.

https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-tpl-function

Reply:
I have taken the bitnami redis chart (which we are using as example) and the rules are rendered as expected 
https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/prometheusrule.yaml#L25